addon/actions/action.js

```diff
@@ -1,19 +1,19 @@
+import { readOnly } from '@ember/object/computed';
+import RSVP from 'rsvp';
+import { assert } from '@ember/debug';
+import { isArray } from '@ember/array';
+import { getOwner } from '@ember/application';
+import ArrayProxy from '@ember/array/proxy';
+import ObjectProxy from '@ember/object/proxy';
+import PromiseProxyMixin from '@ember/object/promise-proxy-mixin';
+import EmberObject, { computed } from '@ember/object';
+import { typeOf as emberTypeOf } from '@ember/utils';
 import Ember from 'ember';
 import deepMerge from 'lodash/merge';
 import normalizePayload from '../utils/normalize-payload';
 import urlBuilder from 'ember-custom-actions/utils/url-builder';
 
 const {
-  RSVP,
-  assert,
-  isArray,
-  getOwner,
-  computed,
-  ArrayProxy,
-  ObjectProxy,
-  PromiseProxyMixin,
-  Object: EmberObject,
-  typeOf: emberTypeOf,
   String: EmberString
```

Comment:
Consider using camelize directly via proper import:

```javascript
import { camelize } from '@ember/string';
```

Reference:
https://github.com/ember-cli/ember-rfc176-data

Reply:
It wasn't available that time ;) thx